pkg/config/config.go

```diff
@@ -212,6 +217,26 @@ func New(opts ...Option) (*Config, error) {
 		}
 	}
 
+	// log the resolved config paths
+	absoluteConfigPaths := make([]string, len(c.paths))
+	for i, path := range c.paths {
+		absoluteConfigPaths[i], err = filepath.Abs(path)
+		if err != nil {
+			absoluteConfigPaths[i] = path
+		}
+	}
```

Comment:
I don't think this is needed as we are replacing paths with abs paths in 119

Reply:
whoops, I think this is residual from a merge, will remove this.